helmfile/values/ambassador.yaml.gotmpl

```diff
@@ -0,0 +1,9 @@
+service:
+  type: ClusterIP
+hostNetwork: true
+dnsPolicy: ClusterFirstWithHostNet
+daemonSet: true
+
+security:
+  podSecurityContext:
+    runAsUser: 0
```

Comment:
Does it need to do this? I think this was changed for the binding of port 80. But in the end port 8080 is used?

Reply:
it is still needed for port 443 right? Which we still need to bind.

Reply:
Didn't it use 8443 for that?

Reply:
no we still have to create a "module" overwriting the default 8443 with 443.

We discussed using the hostPort in the svc instead but I did not manage to get that working

Reply:
I'm convinced this should work (since port 8080 worked) and if it can make it not run as root I think it's time well spent. But I guess it depends how much we should polish this.
Is there anyone that actually needs this or is it just so we can claim that we support it?